src/components/LoginModal/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+import { Modal, ModalVariant, Button } from '@patternfly/react-core';
+
+export default props => {
+  const [isModalOpen, setOpenModal] = useState(false);
+  useEffect(() => {
+    const { showLoginModal } = props;
+    setOpenModal(showLoginModal);
+  });
+  const { redirect, content, callback, modalTitle } = props;
+  const handleModalToggle = () => {
+    setOpenModal(false);
+    callback();
```

Comment:
Nice use of hooks! Can we consider a more descriptive name for our callback function to improve readability? In this case, our callback references `this.modalConfirm()`. What if we considered a naming schema with the `on` prefix (in this case `onConfirm`)?